app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+
+  def create
+      @post = Post.find(params[:post_id])
+      @comment = @post.comments.create(comment_params)
+      redirect_to post_path(@post)
+  end
+
+  def destroy
+      @post = Post.find(params[:post_id])
+      @comment = @post.comments.find(params[:id])
```

Comment:
:+1: Excellent! Can you think of a way to only allow the user who created the post to delete it?